client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -141,36 +141,36 @@ public RssShuffleManager(SparkConf sparkConf, boolean isDriver) {
     this.sparkConf = sparkConf;
 
     // set & check replica config
-    this.dataReplica = sparkConf.getInt(RssSparkConfig.RSS_DATA_REPLICA,
-        RssSparkConfig.RSS_DATA_REPLICA_DEFAULT_VALUE);
-    this.dataReplicaWrite =  sparkConf.getInt(RssSparkConfig.RSS_DATA_REPLICA_WRITE,
-        RssSparkConfig.RSS_DATA_REPLICA_WRITE_DEFAULT_VALUE);
-    this.dataReplicaRead =  sparkConf.getInt(RssSparkConfig.RSS_DATA_REPLICA_READ,
-        RssSparkConfig.RSS_DATA_REPLICA_READ_DEFAULT_VALUE);
-    this.dataTransferPoolSize = sparkConf.getInt(RssSparkConfig.RSS_DATA_TRANSFER_POOL_SIZE,
-        RssSparkConfig.RSS_DATA_TRANSFER_POOL_SIZE_DEFAULT_VALUE);
-    this.dataReplicaSkipEnabled = sparkConf.getBoolean(RssSparkConfig.RSS_DATA_REPLICA_SKIP_ENABLED,
-        RssSparkConfig.RSS_DATA_REPLICA_SKIP_ENABLED_DEFAULT_VALUE);
+    this.dataReplica = sparkConf.getInt(RssSparkConfig.RSS_DATA_REPLICA.key(),
+        RssSparkConfig.RSS_DATA_REPLICA.defaultValue().get());
```

Comment:
```
this.dataReplica = sparkConf.getInt(RssSparkConfig.RSS_DATA_REPLICA);
```
It's more clear.

Reply:
Yes, many MR classes have this kind of code that can be simplified, but the main purpose of the current PR is to reconstruct the parameters of the client. Maybe we can propose a new PR to specifically solve this problem.

Reply:
MR client use class `Configuration`,  the class `Configuration` don't support the parameter `ConfigEntry`.

Reply:
Sorry, we should use the code
```
this.dataReplica = sparkConf.get(RssSparkConfig.RSS_DATA_REPLICA);
```

Reply:
`sparkConf` can only support parameters of string type. For other types such as int, two parameters must be passed in.

Reply:
You can use method `SparkConf.get(ConfigEntry entry)` instead of  method `SparkConf.getInt(String key)`.

Reply:
done.